dockerizeit/master/globalconfig.groovy

```diff
@@ -30,6 +30,14 @@ if (properties.global.jenkinsRootUrl) {
 }
 jlc.save()
 
+// Set Admin Email as a string "Name <email@>"
+println "--> Setting Admin e-mail address"
+if (properties.global.jenkinsAdminEmail) {
+  def jlc = JenkinsLocationConfiguration.get()
+  jlc.setAdminAddress(properties.global.jenkinsAdminEmail)
```

Comment:
Please move print to inside if statement and print actual value together with the message above
Something like
Setting admin e-mail to ....